src/pages/developers/tutorials.tsx

```diff
@@ -0,0 +1,474 @@
+import { useEffect, useMemo, useState } from "react"
+import { GetStaticProps } from "next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { useTranslation } from "react-i18next"
+import { FaGithub, FaYandexInternational } from "react-icons/fa"
+import { Badge, Box, chakra, Flex, forwardRef, Heading, useToken } from "@chakra-ui/react"
+
+import { Lang } from "@/lib/types"
+
+import { Button, ButtonLink } from "@/components/Buttons"
+import Emoji from "@/components/Emoji"
+import FeedbackCard from "@/components/FeedbackCard"
+import InlineLink, { BaseLink } from "@/components/Link"
+import Modal from "@/components/Modal"
+import OldHeading from "@/components/OldHeading"
+import Text from "@/components/OldText"
+import PageMetadata from "@/components/PageMetadata"
+import Translation from "@/components/Translation"
+import { getSkillTranslationId,Skill } from "@/components/TutorialMetadata"
+import TutorialTags from "@/components/TutorialTags"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import { getTutorialsData } from "@/lib/utils/md"
+import { getLocaleTimestamp,INVALID_DATETIME } from "@/lib/utils/time"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+import { filterTutorialsByLang, getSortedTutorialTagsForLang } from "@/lib/utils/tutorial"
+
+import externalTutorials from "@/data/externalTutorials.json"
+
+const FilterTag = forwardRef<{ isActive: boolean; name: string }, "button">(
+  (props, ref) => {
+    const { isActive, name, ...rest } = props
+    return (
+      <chakra.button
+        ref={ref}
+        bg="none"
+        bgImage="radial-gradient(46.28% 66.31% at 66.95% 58.35%,rgba(127, 127, 213, 0.2) 0%,rgba(134, 168, 231, 0.2) 50%,rgba(145, 234, 228, 0.2) 100%)"
+        border="1px"
+        borderColor={isActive ? "primary300" : "white800"}
+        borderRadius="base"
+        boxShadow={!isActive ? "table" : undefined}
+        color="text"
+        fontSize="sm"
+        lineHeight={1.2}
+        opacity={isActive ? 1 : 0.7}
+        p={2}
+        textTransform="uppercase"
+        _hover={{
+          color: "primary.base",
+          borderColor: "text200",
+          opacity: "1",
+        }}
+        {...rest}
+      >
+        {name}
+      </chakra.button>
+    )
+  }
+)
+
+export const getStaticProps: GetStaticProps = async (
```

Comment:
Use the `satisfies GetStaticProps` form
```suggestion
export const getStaticProps = async (
```